src/main/kotlin/com/autonomousapps/DependencyAnalysisPlugin.kt

```diff
@@ -486,7 +492,9 @@ class DependencyAnalysisPlugin : Plugin<Project> {
 
       adviceReport.set(outputPaths.advicePath)
       advicePrettyReport.set(outputPaths.advicePrettyPath)
-      adviceConsoleReport.set(outputPaths.adviceConsolePath)
+      adviceConsoleReport.set(adviceTransformTask.flatMap { it.adviceConsoleReport })
```

Comment:
I don't think this works. I think this would force gradle to create a task graph where `adviceTransformTask` had to run _before_ `adviceTask`. This of course would be impossible given the `finalizedBy`.

I think I understand your earlier concern now. What you want is to just register the transform task above:
```kotlin
val adviceTransformTask = tasks.register<AdviceTransformTask>("adviceTransform$variantTaskName")
```
and then below `adviceTask`
```kotlin
adviceTransformTask.configure { ... }
```